packages/api/src/rooms/usecases/find-room.usecase.ts

```diff
@@ -8,11 +8,11 @@ import { RoomsRepository } from '@/rooms/rooms.repository';
 import { Injectable } from '@nestjs/common';
 
 @Injectable()
-export class FindRoomByNameUsecase implements Usecase {
+export class FindRoomUsecase implements Usecase {
```

Comment:
why the renames? the usecase was supposed to be specific

Reply:
At the moment we dont need to find a room by name so the `FindRoomByNameUsecase` was refactored to find room by ID and renamed to `FindRoomUsecase`

Reply:
ok then 👍